android/app/src/main/res/values-ro/strings.xml

```diff
@@ -552,7 +552,7 @@
 	<!-- Settings: "Send general feedback" button -->
 	<string name="feedback_general">Părere generală</string>
 	<string name="prefs_languages_information">Pentru instrucțiuni vocale utilizăm sistemul TTS. Multe dispozitive Android folosesc Google TTS. Poți descărca sau actualiza aplicația din Google Play (https://play.google.com/store/apps/details?id=com.google.android.tts)</string>
-	<string name="prefs_languages_information_off">Pentru unele limbi trebuie să instalezi alt sintetizator de voce sau un pachet lingvistic suplimentar din Magazinul de aplicații (Google Play, Galaxy Store, App Gallery, FDroid).\nDeschide setările dispozitivului → Limbă → Transformare text în vorbire → Motor preferat.\nAici poți alege motorul de transformare a textului în vorbire și poți descărca o limbă pentru utilizare fără internet.</string>
+	<string name="prefs_languages_information_off">Pentru unele limbi trebuie să instalezi alt sintetizator de voce sau un pachet lingvistic suplimentar din Magazinul de aplicații (Google Play, Galaxy Store, App Gallery, FDroid).\nDeschide Configurări dispozitivului → Limbă → Transformare text în vorbire → Motor preferat.\nAici poți alege motorul de transformare a textului în vorbire și poți descărca o limbă pentru utilizare fără internet.</string>
```

Comment:
Sorry to keep pestering you :-)

I'm not sure about this change. This is for Android. I don't have access to an Android device, so I can't verify how the Setting app is called there, but searching for screenshots on Google, it looks like they call it "Setări". I think it's fine to revert this change and keep it as it was.

Reply:
Really thanks! You're right!